packages/wrangler/src/__tests__/init.test.ts

```diff
@@ -3350,10 +3363,10 @@ describe("init", () => {
 					id: "isolinear-optical-chip",
 				});
 				msw.use(
-					rest.get(
+					http.get(
 						`*/accounts/:accountId/workers/services/:scriptName/environments/:environment/bindings`,
-						(req, res) => {
-							return res.networkError("Mock Network Error");
+						() => {
+							return HttpResponse.error();
```

Comment:
Is `.error()` specifically a network error?

Reply:
Yes—https://developer.mozilla.org/en-US/docs/Web/API/Response/error_static